doc/running.md

```diff
@@ -201,6 +160,150 @@ DJANGO_SETTINGS_MODULE=controlpanel.settings.test pytest
+is documented
+[here](https://github.com/ministryofjustice/analytical-platform-iam#authenticating),
+but the short version is that you need to assume the [restricted-admin@data](https://signin.aws.amazon.com/switchrole?account=mojanalytics&roleName=restricted-admin-data&displayName=restricted-admin@data)
+role (clicking on the link, if you're signed into your AWS dashboard, should do
+this for you). Once you've assumed this role you need to request temporary
+credentials for working with AWS under this role. To do this type the following
+command and read the resulting JSON payload:
+
+```
+aws sts assume-role --role-arn "arn:aws:iam::593291632749:role/restricted-admin-data" --role-session-name data-session
+```
+
+Finally, you need to set the following environment variables, based upon the
+values in the returned JSON payload:
+
+```
+export AWS_ACCESS_KEY_ID=ASIAX........
+export AWS_SECRET_ACCESS_KEY=0vwDrU5..........
+export AWS_SESSION_TOKEN="FQoGZXIvYXdzEHQaDHBH......."
+```
+
+The last step is to check it worked with the following command (which will
+return a JSON payload containing session details):
+
+```
+aws sts get-caller-identity
+```
+
+If you see an error, ask a colleague!
+
+**You probably notice that the session expires after an hour. If you're getting
+AWS errors, ensure you've not timed out.**
+
+### Kubernetes Configuration
+
+For Kubernetes, simply follow the instructions (linked above) for the `dev`
+cluster or set up a local cluster as follows:
+
+To run a Kubernetes cluster locally, the most convenient solution is probably
+`minikube`. Follow the
+[installation instructions for your OS](https://kubernetes.io/docs/tasks/tools/install-minikube/)
+and check you have something working:
+
+```
+$ minikube status
+m01
+host: Running
+kubelet: Running
+apiserver: Running
+kubeconfig: Configured
+```
+
+You'll need a kubeconfig file (`~/.kube/config`) with the credentials for your
+kubernetes cluster. Starting `minikube` (e.g.
+`minikube start --driver=virtualbox`) should create these credentials for you
+for your local machine.
+
+You can check things are working with the following command, which will return
+information about the k8s master and KubeDNS:
+
+```sh
+kubectl cluster-info
+```
+
+#### Helm
+
+Install Helm (the K8s package manager) by following
+[these instructions for your OS](https://helm.sh/docs/intro/install/).
+
+You'll need to initialise Helm too:
+
+```sh
+helm init
+```
+
+Tell Helm to use the Analytical Platform chart repository:
+
+```sh
+helm repo add mojanalytics http://moj-analytics-helm-repo.s3-website-eu-west-1.amazonaws.com
+helm repo update
+```
+
+## Create superuser (on first run only)
+
+This isn't strictly required, so feel free to skip this step.
+
+```sh
+python3 manage.py createsuperuser
+```
+
+Your `Username` needs to be your GitHub username.
+Your `Auth0 id` needs to be the number associated with you in auth0.com and
+labelled `user_id` (not working for me yet).
```

Comment:
maybe tweak to:

> Your `Auth0 id` needs to be the number associated with you in auth0.com and
labelled `user_id` (including the `github|` prefix, e.g. `github|123...`).

If I understand correctly, this is now working fine for you now right?

Reply:
Nope. I don't honestly think this step is required.